src/server/rdb_load.cc

```diff
@@ -2505,6 +2557,11 @@ void RdbLoader::LoadItemsBuffer(DbIndex db_ind, const ItemsBuf& ib) {
       stop_early_ = true;
       break;
     }
+    if (item->load_config.append) {
```

Comment:
why did you add this code?

Reply:
A bit of a guess given it already does `ts->TryStash` below in the non-append case

Reply:
Lets remove this code.
1. Data tiering does not work with to non-strings.
2. We do not want to stash values that are not yet finalized.

Reply:
sure removed